x/multi-staking/types/utils.go

```diff
@@ -4,6 +4,14 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+const (
+	// BaseDenom defines the base denomination
+	BaseDenom = "ario"
```

Comment:
should be `MultistakingDenomA` and `MultistakingDenomB`. also should be in testing package.